cmd/libs/go2idl/client-gen/README.md

```diff
@@ -0,0 +1,23 @@
+Client-gen is an automatic tool that generates [clientset](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/client-package-structure.md#high-level-client-sets) based on API types. This doc introduces the use the client-gen, and the release cycle of the generated clientsets.
+
+## Using client-gen
+
+The workflow include four steps:
+- Marking API types with tags: in `pkg/apis/${GROUP}/${VERSION}/types.go`, mark the types (e.g., Pods) that you want to generate clients for with the `// +genclient=true` tag.  If the resource associated with the type is not namespace scoped (e.g., PersistentVolume), you need to append the `nonNamespaced=true` tag as well.
```

Comment:
I think that's fine for a future PR. I'm just trying to get the docs for the current tool right. An example --input flag would be appreciate.